src/validation.h

```diff
@@ -228,7 +228,7 @@ bool TestLockPointValidity(const LockPoints* lp) EXCLUSIVE_LOCKS_REQUIRED(cs_mai
  *
  * See consensus/consensus.h for flag definitions.
  */
-bool CheckSequenceLocks(const CTxMemPool& pool, const CTransaction& tx, int flags, LockPoints* lp = nullptr, bool useExistingLockPoints = false) EXCLUSIVE_LOCKS_REQUIRED(::cs_main, pool.cs);
+bool CheckSequenceLocks(CChainState& active_chainstate, const CTxMemPool& pool, const CTransaction& tx, int flags, LockPoints* lp = nullptr, bool useExistingLockPoints = false) EXCLUSIVE_LOCKS_REQUIRED(::cs_main, pool.cs);
```

Comment:
I find these 200+ character lines very difficult to parse. It'd be nice if PRs didn't make them even longer and more unreadable. Consider something like:

```suggestion
bool CheckSequenceLocks(CChainState& active_chainstate, const CTxMemPool& pool,
                        const CTransaction& tx, int flags, LockPoints* lp = nullptr,
                        bool useExistingLockPoints = false)
    EXCLUSIVE_LOCKS_REQUIRED(::cs_main, pool.cs);
```

(same for the function definition in validation.cpp)

Reply:
Done in: bf2927643162c650f2a54dd33a83eccaa67ac50c